docs/en/stack/ml/anomaly-detection/ml-configuration.asciidoc

```diff
@@ -18,9 +18,11 @@ you visualize and explore the results.
 
 * <<create-jobs>>
 * <<stopping-ml>>
+* <<ml-recover-failed-jobs>>
 
-After you learn how to create and stop {anomaly-detect} jobs, you can check the 
-<<anomaly-examples>> for more advanced settings and scenarios.
+After you learn how to create and stop {anomaly-detect} jobs, and how to recover 
+from a failed job, you can check the <<anomaly-examples>> for more advanced 
+settings and scenarios.
```

Comment:
I am not sure about mentioning recovering from failed jobs in this sentence.  It makes it sound like it's one of the first things you have to learn when it's not really.  It's just something that needs to be easy to find out about _if_ you do need it.